os/drivers/lcd/mipi_lcd.c

```diff
@@ -356,7 +362,7 @@ static int lcd_init(FAR struct lcd_dev_s *dev)
 		lcddbg("ERROR: LCD Init sequence failed\n");
 	}
 	priv->config->init();
-	priv->config->lcd_enable();
+	priv->config->backlight(CONFIG_LCD_MAXPOWER);
```

Comment:
This looks not related with logo support. Could you let me know why you change lcd_enable to backlight control?